pkg/consts/consts.go

```diff
@@ -73,6 +75,10 @@ const (
 	// Used by the home chain reader.
 	MethodNameGetAllChainConfigs = "GetAllChainConfigs"
 	MethodNameGetOCRConfig       = "GetOCRConfig"
+
+	// RMNHome.sol methods
+	// Used by the rmn home reader.
+	MethodNameGetAllConfigs = "getAllConfigs"
```

Comment:
```suggestion
	MethodNameGetAllConfigs = "GetAllConfigs"
```

In theory this doesn't have to match the chain-specific name, so lets make it "generic"